### DISCUSSION TITLE
Welcome to my-website Discussions!

### DISCUSSION THREAD

**Discussion by wghglory** [wghglory/my-website#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by davidchen8994** [comment]
Hi @wghglory, What a cool website you're building! I'm learning how to build a personal website now. Your website is quite shining, especially the design and techniques you use. May I copy some code? Thank you! ☕️

**Reply by wghglory** [reply]
Hey @davidchen8994 Please feel free to copy code and use them in your website. It's my pleasure if my website is helpful. I just ran this website and would share posts, snippets from now on. Feel free to leave comments and have discussions. Thanks!

**Reply by davidchen8994** [reply]
Sure thing! I'm looking forward to your posts!
